core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -201,9 +173,9 @@ public void addLostStorage(String tierAlias, String dirPath) {
    */
   public void addLostStorage(Map<String, StorageList> lostStorage) {
     for (Map.Entry<String, StorageList> entry : lostStorage.entrySet()) {
-      List<String> paths = mLostStorage.getOrDefault(entry.getKey(), new ArrayList<>());
+      List<String> paths = mUsage.mLostStorage.getOrDefault(entry.getKey(), new ArrayList<>());
```

Comment:
is this method called with usage-lock held? If so can you update the method comment?